src/components/FeaturedContent/FeaturedContent.tsx

```diff
@@ -43,7 +43,7 @@ export interface FeaturedContentProps {
  */
 function FeaturedContentImage(props: FeaturedContentImageProps) {
   const { alt, src } = props;
-  return <img alt={alt} src={src} style={{ display: "none" }} />;
+  return <Image alt={alt} src={src} style={{ display: "none" }} />;
```

Comment:
@EdwinGuzman Now using the `Image` component, though still needs to have `style={{ display="none }}` as far as I can tell. This isn't related to the `imageProps` warning failing to appear though, which is strange– even if I take the `style={{ display="none }}` off and have `img: screenReaderOnly()`, the test is only catching that `imageProps.alt` and `imageProps.src` aren't there, not that `imageProps` isn't on the component at all.